src/components/Categories.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>
+    <ul style={{
+      display: 'flex',
+      flexDirection: 'row',
+      maxWidth: '100%',
+      padding: '3%',
+    }}
+    >
+      <li style={{
+        color: 'blue',
+        fontWeight: 'bold',
+        fontSize: '1.1em',
+        marginLeft: '4%',
+      }}
+      >
+        Romance
+      </li>
+      <li style={{
+        color: 'blue',
+        fontWeight: 'bold',
+        fontSize: '1.1em',
+        marginLeft: '4%',
+      }}
+      >
+        Science fiction
+      </li>
+      <li style={{
+        color: 'blue',
+        fontWeight: 'bold',
+        fontSize: '1.1em',
+        marginLeft: '4%',
+      }}
+      >
+        Fantasy
+      </li>
+      <li style={{
+        color: 'blue',
+        fontWeight: 'bold',
+        fontSize: '1.1em',
+        marginLeft: '4%',
+      }}
+      >
+        Economy
+      </li>
+    </ul>
+  </div>
```

Comment:
As required, `Categories.js` should display "Under construction" text only

![image](https://user-images.githubusercontent.com/66112760/152398224-2be2995d-e374-4bbc-9e6a-d098784697b4.png)

Kindly fix this 👍

Reply:
Please what does it mean?